dialogue-core/src/main/metrics/dialogue-core-metrics.yml

```diff
@@ -23,3 +23,7 @@ namespaces:
         type: meter
         tags: [service-name]
         docs: Rate of deprecated endpoints being invoked.
+      limited:
+        type: meter
+        tags: [reason]
+        docs: Rate of client-side request rejection.
```

Comment:
We might need to tweak this a little as it sounds quite scary, especially if people don't know about the retries or queue.